Makefile

```diff
@@ -240,10 +245,12 @@ $(YQ): $(LOCALBIN)
 	test -s $(LOCALBIN)/yq || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@latest
 
 .PHONY: bundle
-bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+bundle: manifests kustomize yq ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
+	$(YQ) eval '(.spec.install.spec.deployments[].spec.template.spec.containers[].env.[] | select(.name == "SANDBOXED_CONTAINERS_EXTENSION") | .value) |= "$(SANDBOXED_CONTAINERS_EXTENSION)"' \
+		-i bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml
```

Comment:
I'd actually prefer doing the replacement in `config/manager/manager.yaml` before kustomizing instead of in the generated file after kustomizing since the difference between the template and the resulting generated file will end up konfusing someone I'm afraid.  But this works too.

Reply:
If the replacement is done in `config/manager/manager.yaml` then it will be dirty... which basically negates the benefit of this commit or I am missing something ?

Reply:
There would still be benefit in setting this in `Makefile` and not having to know or find out where exactly you as a (new) developer need to make this change.  As I said, I'm fine with this way, too.

Reply:
On the confusion side : people still need to explicitly set `SANDBOXED_CONTAINERS_EXTENSION`  to something != from the default value (`kata-containers`) to change the default behavior. They shouldn't be that much surprised.